docs/guides/operations-guide/manager/apply.md

```diff
@@ -34,3 +38,56 @@ $ osism apply
 | iscsi                                                            | kolla          |
 ...
 ```
+
+## Apply a role
+
+```
+$ osism apply operator -l st01-mgmt-r01-u30
+2024-06-14 09:33:10 | INFO     | Task f94a2e6f-d199-421c-b7b7-743db4661305 (operator) was prepared for execution.
+2024-06-14 09:33:10 | INFO     | It takes a moment until task f94a2e6f-d199-421c-b7b7-743db4661305 (operator) has been started and output is visible here.
+
+PLAY [Make ssh pipelining working] *********************************************
+
+TASK [Do not require tty for all users] ****************************************
+ok: [st01-mgmt-r01-u30]
+
+...
+...
+
+TASK [osism.commons.operator : Set password of operator user] ******************
+ok: [st01-mgmt-r01-u30]
+
+TASK [osism.commons.operator : Set root password like operator password] *******
+ok: [st01-mgmt-r01-u30]
```

Comment:
This task is not part of the operator role and the PR will not be merged. We added the password feature in the osism.commons.user role.

Reply:
Removed it